modules/configuration/ajax/process.php

```diff
@@ -60,7 +60,7 @@
                 }
             }
             // Update the config setting to the new value.
-            $DB->update(
+            $DB->unsafeUpdate(
```

Comment:
is this okay to change for all update/inserts on the config module, not knowing what other front-end values this might affect?

Reply:
I looked on demo and only saw 3 textarea config settings:

1. The studyDescription used on the login page. This already uses DOMPurify to make sure the value is clean.
2. The projectDescription used by the dashboard welcome widget. An admin would be able update the description to include unsafe HTML there, but this is addressed by PR #8762. 
3. The citation_policy used by the acknowledgements module. This is used by the acknowledgements module inside of a React component, so it should be auto-escaped by React.